src/main/constants.js

```diff
@@ -3,9 +3,5 @@ export const LASTFM_API_SECRET = process.env.GPMDP_LASTFM_API_SECRET;
 
 if (!LASTFM_API_KEY || !LASTFM_API_SECRET ||
   LASTFM_API_KEY === 'undefined' || LASTFM_API_SECRET === 'undefined') {
-  console.error('####################################'); // eslint-disable-line
-  console.error('############### ERROR ##############'); // eslint-disable-line
-  console.error('######### LASTFM  API KEYS #########'); // eslint-disable-line
-  console.error('######## HAVE NOT BEEN SET #########'); // eslint-disable-line
-  console.error('#####################################'); // eslint-disable-line
+  Logger.error('Last.fm API keys have not been set!');
```

Comment:
I would want this to go to the console aswell, not sure if that is how winston works or not